src/main/client/type.c

```diff
@@ -537,6 +538,19 @@ static PyObject *AerospikeClient_Type_New(PyTypeObject *type, PyObject *args,
     return (PyObject *)self;
 }
 
+#define MAX_ERR_MSG_SIZE 250
+
+static void initialize_config_value_type_err(
+    char error_msg[MAX_ERR_MSG_SIZE],
+    // config_keys should look like ["lua"]["user_path"]
+    const char *config_keys, const char *expected_type_name,
+    int *error_code_ptr)
+{
+    snprintf(error_msg, MAX_ERR_MSG_SIZE, "config%s must be a %s", config_keys,
+             expected_type_name);
```

Comment:
```suggestion
    snprintf(error_msg, MAX_ERR_MSG_SIZE, "config %s must be a %s", config_keys,
             expected_type_name);
```

spacing between "config" and the config name